src/GCRealTimeMon/Configuration/ConfigurationReader.cs

```diff
@@ -18,12 +20,17 @@ internal static class ConfigurationReader
         /// <exception cref="ArgumentNullException"></exception>
         public static async Task<Configuration> ReadConfigurationAsync(string path)
         {
-            IDeserializer deserialier = new DeserializerBuilder()
+            if (string.IsNullOrEmpty(path))
```

Comment:
The path can never be null + ArgumentNullException is not properly handled in calling code